pqlite/index.py

```diff
@@ -82,22 +85,27 @@ def __init__(
 
         self.vq_codec = None
         if self._vq_codec_path.exists() and n_cells > 1:
-            logger.info(
-                f'Load trained VQ codec (K={self.n_cells}) from {self.model_path}'
-            )
+
+            if self.logger:
+                logger.info(
+                    f'Load trained VQ codec (K={self.n_cells}) from {self.model_path}'
+                )
```

Comment:
If you don't want report this message. we can change the message level via `logger.debug()` instead.